.github/workflows/register.yml

```diff
@@ -2,6 +2,8 @@ name: 🐳 Register
 
 on:
   push:
+    tags:
+      - '*'
```

Comment:
Might not be needed as any commit can might be deployed on the Kube target. 
The registry workflow tags commit sha, branches and tags